src/Dfe.PlanTech.Web/Controllers/CacheController.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Dfe.PlanTech.Web.Helpers;
+using EFCoreSecondLevelCacheInterceptor;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Dfe.PlanTech.Web.Controllers;
+
+[Route("cache")]
+public class CacheController : BaseController<CacheController>
+{
+    public CacheController(ILogger<CacheController> logger) : base(logger)
+    {
+    }
+
+    [HttpPost("clear-cache")]
+    [ValidateApiKey]
+    public bool ClearCache([FromServices] IEFCacheServiceProvider cacheServiceProvider)
+    {
+        try
+        {
+            cacheServiceProvider.ClearAllCachedEntries();
+            logger.LogInformation("Database cache has been cleared");
+            return true;
+        }
+        catch (Exception e)
+        {
+            logger.LogError("An error occured while trying to clear the database cache: {message}", e.Message);
```

Comment:
Ideally the exception should be logged here. E.g.

`logger.LogError(e, "An error occured while trying to clear the database cache")`

or

`logger.LogError(e, "An error occured while trying to clear the database cache: {message}", e.Message);`

Reply:
Ahh, I didn't realise this was a convention - updated 👍 thanks